### PR TITLE
Add the AddMultiple and AddMultipleUnion methods to Go2TS.

### PR DESCRIPTION
**Pull request by jcgregorio**

### CODE REVIEW

File: go2ts.go [hunk 1]
```diff
@@ -43,6 +43,18 @@ func (g *Go2TS) Add(v interface{}) error {
 	return g.AddWithName(v, "")
 }
 
+// AddMultiple types in a single call.
```
Comment (LeandroLovisolo) [inline go2ts.go]: // AddMultiple **adds multiple** types in a single call.

File: go2ts_test.go [hunk 2]
```diff
@@ -255,6 +255,92 @@ export interface ADifferentName {
...
+
+export interface AnotherStruct {
+	A: string;
+}
+`
+	assert.Equal(t, expected, b.String())
+}
+
+func TestRender_AddMultipleEmpty_Success(t *testing.T) {
+
```
Comment (LeandroLovisolo) [inline go2ts_test.go]: Nit: Please remove extra blank line.

**Review by LeandroLovisolo — APPROVED** [review]
Thanks! Just two nits.

File: go2ts.go (hunk 1 above)
Comment (jcgregorio) [reply go2ts.go]: Done.

File: go2ts_test.go (hunk 2 above)
Comment (jcgregorio) [reply go2ts_test.go]: Done.

**Pull request merged**